### ISSUE TITLE
Small Confusion About Plugin Options

### ISSUE THREAD

**Issue by RomanScott** [getsentry/sentry-webpack-plugin#112]
Hello,

I wish to integrate Sentry into a project I have been working on, as I believe it is a wonderful error-tracking tool! Currently, I am using the Sentry Electron SDK and am packaging it into an executable using Webpack (I am using [Electron-Vue](https://github.com/SimulatedGREG/electron-vue) to accomplish this process.)

My file structure looks like this:

```
electron-vue
     webpack.main.config.js
     webpack.renderer.config.js
     build.js

build/icons/icon.ico

node_modules

src
      main
            <other source code>
            app.js 
      renderer
            App.Vue
            renderer.js

static
      res/img

.babelrc
.travis.yml
package.json
yarn.lock
```

I am confused as to what I should "include." I have two webpack configs, each of which builds a separate part of the app and outputs a main.js and a renderer.js (for the renderer JavaScript portion of the app which is written and compiled with Vue.)

Do I only include those specific folders, like in webpack.main.config I include src/main only and then in webpack.renderer.config I include src/renderer? Or do I have to do `include: '.'`?

In other words, do I need stuff like node_modules and everything else also included?

A few more questions:
What specifically does "entries" do, and for "ignore" is it just ignoring files during the upload process? Does that mean node_modules needs to be "included" and "ignored?" Why is node_modules included at all?

Sorry for the long issue, I just wanted to clarify. Thank you kindly.

**Comment by kamilogorek** [comment]
> Do I only include those specific folders, like in webpack.main.config I include src/main only and then in webpack.renderer.config I include src/renderer? Or do I have to do include: '.'?
> In other words, do I need stuff like node_modules and everything else also included?

You should only include a directory that contains all your build artifacts and source maps (basically everything from your `output` in the webpack config).

> What specifically does "entries" do

It's a list of files that we'll stick the code above to:
```js
const sentryRelease = `(typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {}).SENTRY_RELEASE={id:"${version}"};`;
```

(https://github.com/getsentry/sentry-webpack-plugin/blob/master/src/sentry.loader.js)

> for "ignore" is it just ignoring files during the upload process?

Yes, just files.

> Does that mean node_modules needs to be "included" and "ignored?" Why is node_modules included at all?

If your `include` config doesn't contain a directory that's the parent of `node_modules`, it won't be uploaded, so you can skip it.

It's included because in the example we are including the root folder `"."` which contains `node_modules`, so CLI would go and traverse through it as well, which we don't want. Ignoring `node_modules` is in 99.9% desirable so it's a default anyway.

**Comment by kamilogorek** [comment]
Closing due to inactivity
